templates/DynamicsCrm/CardDynamicsCrm.html

```diff
@@ -42,6 +42,5 @@
         </tbody>
       </table>
     </div>
-    <div class="CoveoFollowItem"></div>
```

Comment:
Just want to confirm that you manually removed this component from the html files? If so, where is it now?

Reply:
It's a component that is available for clients to use, it's just that for most client, the micro service that needs to be activated for this component to work is not activated 90% of the time.

It's on a per demand basis that the service will be activated for specific clients. So I'm just removing it from the sample/default files so it's not always included and tries to connect/pinge the service for no reason each time.

It can still be added manually on a per implementation basis if they need it, it's just not the majority of clients that will use/need this.